examples/example-gateway/gateway.json

```diff
@@ -7,5 +7,6 @@
 
 	"clientConfig": "./clients",
 	"endpointConfig": "./endpoints",
-	"middlewareConfig": "./middlewares/middleware-config.json"
+	"middlewareConfig": "./middlewares/middleware-config.json",
+	"copyrightHeader": "./copyright_header.txt"
```

Comment:
I want whoever's using zanzibar to generate gateways to be able to attach their own copyright header, since the copyright for the generated code should be attached to whoever defines the specification (but not fro the runtime or generators)